### PR TITLE
updates README and usage instructions

### PR DESCRIPTION
**Pull request by joemfb**
This just removes/replaces anything that's actively wrong. The help text is still incomplete, and could definitely be improved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,34 +72,11 @@ Once the project is configured, use `ninja` to build it.
...
-+ Boot up a ship using the `urbit` command.
-
 ## Contact
 
 If you have any questions, problems, patches, or proposals for patches, please
 feel free to get in touch in whatever way is most convenient:
 
-- Post to `/urbit-meta` on Urbit `:talk`.  (You can do this via
-[urbit.org/stream](https://urbit.org/stream) without a running Urbit).
+- Post to `/urbit-meta` on Urbit [`:talk`](https://urbit.org/docs/learn/arvo/arvo-internals/messaging/).
```
Comment (joemfb) [inline README.md]: This new link is not currently correct, but I believe it will be shortly.

File: vere/main.c [hunk 2]
```diff
@@ -388,8 +388,8 @@ u3_ve_usage(c3_i argc, c3_c** argv)
     "   https://github.com/urbit/urbit/blob/master/CONTRIBUTING.md\n",
     "\n",
     "Simple Usage: \n",
-    "   %s -c <mycomet> to create a comet (anonymous urbit)\n",
-    "   %s -w <myplanet> -t <myticket> if you have a ticket\n",
+    "   %s -c <my-comet> to create a comet (anonymous urbit)\n",
+    "   %s -w <my-planet> -k <my-key-file> if you own a planet\n",
```
Comment (belisarius222) [inline vere/main.c]: `if you own a planet or other Azimuth point`

Comment (joemfb) [reply vere/main.c]: See also `<my-planet>`, "restart your planet/comet" ...

File: README.md (hunk 1 above)
Comment (vvisigoth) [reply README.md]: @msutherl are we changing the "help" channel to /urbit-help?

File: vere/main.c (hunk 2 above)
Comment (belisarius222) [reply vere/main.c]: I don't understand what you mean

**Comment by joemfb** [comment]
The usage text should make a further distinction between *development* and *usage*. For example, the fake ~zod instructions are for developing on something other than master. To just boot a fake ship (of any address class), simply `urbit -F ship-name`.

**Comment by joemfb** [comment]
Any class except a comet, that is. There's really no such thing as a fake comet.

File: vere/main.c (hunk 2 above)
Comment (belisarius222) [reply vere/main.c]: I see. We can leave it as planet, then

**Comment by joemfb** [comment]
A comet is the fingerprint of the combined public key (`+pass`) of a `+crub` key-pair. Fake-ship keys are derived directly from the ship itself as a seed (treated as a 512 bit uint). It may be possible to square this circle, but I don't see how. And it doesn't strike me as urgent.

**Pull request merged**